.github/workflows/test-multiple-builds.yml

```diff
@@ -37,11 +37,11 @@ jobs:
       - name: Patch for CJS
         if: ${{ matrix.build == 'cjs' }}
         run: |
-          sed -i~ "s/resolve('\.\/src\(.*\)\.ts')/resolve('\.\/dist\1.js')/" vitest.config.ts
+          sed -i~ "s/resolve('\.\/src\(.*\)\.ts')/resolve('\.\/dist\1.js')/" vitest.config.mts
```

Comment:
Is this change necessary for this PR?
Can you open a separate PR and open the same PRs to `jotai` and `valtio` too?

Reply:
@dai-shi sure, let me create a new PR for that